docs/new-architecture-app-modules-android.md

```diff
@@ -9,6 +9,12 @@ import constants from '@site/core/TabsConstants';
 
 <NewArchitectureWarning/>
 
+:::info
+For the sake of this guide we're going to use the following **terminology**:
+
+- **Turbo Native Modules** - To refer to Modules which have been adapted to work well with the New Native Module System. For brevity you might find them referred as **Turbo Modules**
+  :::
+
```

Comment:
> I personally find bullet lists with a single bullet a bit odd.

Yup. I removed all the boxed which had only one bullet point 👍 

> should we put this banner on top of every page where we use this terminology

I'm a bit hesitant on adding yet another banner as it would bloat the page I think. I would stick to having this notice only on pages where we use both Legacy and Fabric|Turbo terminology